vignettes/add_palettes.Rmd

```diff
@@ -19,5 +19,5 @@ knitr::opts_chunk$set(
 Fargepaletten til PwC er vel og bra, men veldig ofte lager vi figurer for kunde som må følge deres fargekoder. Hex-koder er noe styr å jobbe med, så denne vignetten viser hvordan man kan legge til fargene til klienter i denne pakken. På sikt bør vi ha samlet fargepalettene til mange av våre kunder i denne pakken, slik at det er enkelt å lage flotte figurer i riktige farger. 
 
 ```{r setup}
-library(pwccolorrr)
+#library(pwccolorrr)
```

Comment:
Dette tror jeg er et arbeidsnavn på pakken - kan sikkert fjernes

Reply:
Tenker resten kan stå der. Måtte bare kommentere det ut for å kunne dokumentere/bygge. Dette er arvesynd som dessverre kommer med i denne PRen